### ISSUE TITLE
Script is still spreading when there is no fire

### ISSUE THREAD

**Issue by Sythy11** [Pyth3rEx/esx_fireDispatch#8]
So sometimes I have noticed that after some fires, the script will still be trying to spread the fire. When looking at F8, it's just constantly spreading...  spreading...   But it's only after some fires.

**Comment by Pyth3rEx** [comment]
I think I know why that is, I'll try and fix that for the next update!

**Comment by Sythy11** [comment]
I'm glad you are still working on this project. We are very interested to get it working with our server.

Also another bug, is ALL firestations alarm's go off, not just the closest to the fire. 

And im not 100% sure of the random timer.

**Comment by Pyth3rEx** [comment]
So:
1- thank you very much ^^
2- So this is a 'bug exploit'. Making players believe all stations do the alarm (which is the wanted effect) but really it does the alarm on the station closest to the player.
3- And the timer is not random at all. It's configured in the config. (it's a temporary fix)

**Comment by Sythy11** [comment]
randomStart = true,
randomTime = 50000, 

I know i've done it before, but I've made the number a lot smaller, to the point the random should be spawning a fire almost every second. But it's not spawning them at all.

**Comment by Pyth3rEx** [comment]
Mind posting another issue for that? Just for me to stay organised ^^
Thanks

**Comment by Sythy11** [comment]
No Problem

**Comment by Sythy11** [comment]
https://puu.sh/Eqye7/751dcb6a97.jpg

This would happen to be the problem as to why it is still spreading. Technically the fire isn't actually out, but it also isn't accessible by players.

**Comment by Pyth3rEx** [comment]
ah... not sure if I'll ever be able to fix that tbh...

**Comment by Sythy11** [comment]
I see there is a 'if ground' section, but obviously the game recognizes the ground there. Even if a player would fall through the map. 

It restricts where we can place fires unfortunately.

**Comment by Pyth3rEx** [comment]
Also fixing #10 would probably help lol
